common/scripted_triggers/hfe_trigger.txt

```diff
@@ -18,6 +18,7 @@ hfe_trigger_uncapped_job = {
 		has_job = hfe_job_future_workless
 		has_job = hfe_job_brainworm_reservation_paternalist
 		has_job = hfe_job_pirate_slave
+		has_job = hfe_job_godforge_fulfilment_center
```

Comment:
this triggers not needed anymore

Reply:
No need to remove it. I've seen it called before.